toolkit/scripts/srpm_expand.mk

```diff
@@ -37,7 +37,7 @@ $(STATUS_FLAGS_DIR)/build_specs.flag: $(srpms) $(BUILD_SRPMS_DIR)
 		srpm_file=$(BUILD_SRPMS_DIR)/$${srpm} && \
 		spec_destination=$(BUILD_SPECS_DIR)/$$(rpm -qp $$srpm_file --define='with_check 1' --queryformat %{NAME}-%{VERSION}-%{RELEASE}/%{NAME}.spec) && \
 		spec_dir=$$(dirname $$spec_destination) && \
-		if [ ! -f $@ -o $$srpm -nt $@ ]; then \
+		if [ ! -f $@ -o $$srpm_file -nt $@ ]; then \
```

Comment:
What do you think about removing `srpms_basename` and `srpm_file`, and doing `for srpm in $(srpms); do` instead? Right now it looks like we're converting `srpms` with full paths to the SRPMs into the base names just to convert them back into a full-pathed `srpm_file`.

Reply:
Good point- I considered that, but in the past I've run into issues using the full srpm filename in the for loop invocation. Something about `sh` not liking the argument length. See: #709

More than happy to look and see if that's still a problem, but for now I'd like to just fix this one issue.